docs/index.bs

```diff
@@ -2181,62 +2157,64 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
+                        1. Resolve |promise| with the result of running the algorithm specified in {{Cache/match(request, options)}} method of {{Cache}} interface with |request| and |options| (providing |cache| as thisArgument to the <a>\[[Call]]</a> internal method of {{Cache/match(request, options)}}.)
                         1. Abort these steps.
-                1. Resolve |p| with undefined.
+                1. Resolve |promise| with undefined.
         1. Else:
-            1. Let |p| be a <a>promise</a> resolved with undefined.
-            1. [=map/For each=] <var ignore>cacheName</var> → |cache| of <a>name to cache map</a>:
-                1. Set |p| to the result of <a>transforming</a> itself with a fulfillment handler that, when called with argument |v|, performs the following substeps <a>in parallel</a>:
-                    1. If |v| is not undefined, return |v|.
-                    1. Return the result of running the algorithm specified in {{Cache/match(request, options)}} method of {{Cache}} interface with |request| and |options| as the arguments (providing |cache|] as thisArgument to the <a>\[[Call]]</a> internal method of {{Cache/match(request, options)}}.)
-            1. Return |p|.
+            1. Let |promise| be [=a promise resolved with=] undefined.
+            1. [=map/For each=] <var ignore>cacheName</var> → |cache| of the [=name to cache map=]:
+                1. Set |promise| to the result of [=transforming=] itself with a fulfillment handler that, when called with argument |response|, performs the following substeps [=in parallel=]:
+                    1. If |response| is not undefined, return |response|.
+                    1. Return the result of running the algorithm specified in {{Cache/match(request, options)}} method of {{Cache}} interface with |request| and |options| as the arguments (providing |cache| as thisArgument to the `\[[Call]]` internal method of {{Cache/match(request, options)}}.)
+            1. Return |promise|.
     </section>
 
     <section algorithm="cache-storage-has">
       <h4 id="cache-storage-has">{{CacheStorage/has(cacheName)}}</h4>
 
       <dfn method for="CacheStorage"><code>has(|cacheName|)</code></dfn> method *must* run these steps:
 
-        1. Return a <a>promise</a> resolved with the result of running the following substeps:
-            1. [=map/For each=] |key| → <var ignore>value</var> of <a>name to cache map</a>:
+        1. Let |promise| be [=a new promise=].
+        1. Run the following substeps [=in parallel=]:
+            1. [=map/For each=] |key| → <var ignore>value</var> of the [=name to cache map=]:
                 1. If |cacheName| matches |key|, then:
-                    1. Return true.
-            1. Return false.
+                    1. Resolve |promise| with true.
+                    1. Abort these steps.
```

Comment:
Could probably roll this into the line above. "Resolve promise with true and abort these steps".

Reply:
Addressed.